src/components/StatisticsCard/Damage.js

```diff
@@ -3,7 +3,8 @@ import { OverlayTrigger, Tooltip as Tips } from 'react-bootstrap';
 import LineChart from '../Charts/LineChart';
 
 const Damage = (props) => {
-  const lastSnapshot = props.accountStats.data.snapshots.at(-1);
+  const stats = props.accountStats ? props.accountStats.data : props.tankStatsCard;
```

Comment:
To simplify the code, you can utilize the logical OR (||) operator
```suggestion
  const stats = props.tankStatsCard || props.accountStats?.data;
```